### PR TITLE
Pks/perma link blog pages

### LINKED ISSUE #64: use standard show route redirect to index
Use the standard show route for social linking, but have the controller redirect to the index#id url. Hijack the the back button/action so it redirects to the index.

This will allow us to inject the appropriate meta tags and images for sharing and provide a unique addressable url for each post.

### PR DESCRIPTION
**Pull request by ninjapanzer**
### What am I?
Cleanup some of the mess from recent changes and removals.
- There was a merge .orig left behind
- Footer scroll js handler was blocking pages from scrolling back up once the bottom of the page was hit
- Add post page to directly load a single post

### CODE REVIEW

File: app/assets/javascripts/sub-nav.js [hunk 1]
```diff
@@ -1,10 +0,0 @@
-$(document).ready(function() {
-  $(window).scroll(function () {
-    if ($(window).scrollTop() > 140) {
-      $('.sub-nav-fixed').addClass('navbar-fixed');
-    }
-    if ($(window).scrollTop() < 140) {
-      $('.sub-nav-fixed').removeClass('navbar-fixed');
-    }
-  });
-});
```
Comment (garyanewsome) [inline app/assets/javascripts/sub-nav.js]: This is the scrolling navbar feature we talked about and poked at yesterday, Ima gonna check over everything again make sure everything works.

File: app/views/posts/index.haml [hunk 2]
```diff
@@ -11,24 +11,24 @@
           .gallery-curve-wrapper
             %a.gallery-cover.gray
               = image_tag attachment_url(post.post_attachments.first, :post_image, :fill, 350, 250), class: "responsive-img"
-              / use below after upgrade
-              /= image_tag attachment_url(post.post_attachments.first, :post_image, :fill, 350, 250, fallback: "blog_fallback.png"), class: "responsive-img"
+              -# use below after upgrade
+              -# = image_tag attachment_url(post.post_attachments.first, :post_image, :fill, 350, 250, fallback: "blog_fallback.png"), class: "responsive-img"
```
Comment (garyanewsome) [inline app/views/posts/index.haml]: Is this a new haml style comment-er-out-er

Comment (ninjapanzer) [inline app/views/posts/index.haml]: `/` will be an html comment
`-#` is evaluated as a ruby comment and will not be in the output or even executed. It avoids page errors for non-displaying images

File: app/assets/javascripts/sub-nav.js (hunk 1 above)
Comment (ninjapanzer) [inline app/assets/javascripts/sub-nav.js]: I will test it again

**Comment by garyanewsome** [comment]
I tested it with two different posts seemed to work ok, give it a pull and try again??? /shrug

**Pull request merged**